src/plugins/intel_gpu/src/graph/graph_optimizer/remove_redundant_reorders.cpp

```diff
@@ -554,6 +555,43 @@ void remove_redundant_reorders::run(program& p) {
         }
     }
 
+    // Remove reorder in front of MVN
+    // when the output format of reorder is bfyx and
```

Comment:
Shouldn't we check the output format of `dep_node` is equal to `bfyx`?

Reply:
The purpose of this code is to remove the reorder in front of MVN node.
So, it doesn't need to check the output format of reorder.
But, I think I should add additional conditions for padding because current MVN bsv32 kernel doesn't support spatial padding yet. Thanks.

Reply:
I think we need to ensure that input of reorder is same as output of mvn.